tox.ini

```diff
@@ -1,47 +1,56 @@
 [tox]
 envlist=
-    py{37,38,39,310,311,py3}-core
-    lint
+    py{37,38,39,310,311}-core
+    py{38,39,310,311}-lint
```

Comment:
We should either add the 37-lint here or remove it from `.circleci/config.yml`. As it stands on CI, a 37-lint job "runs" but it doesn't check anything.

Reply:
Removed py37-lint from circleci